### PR TITLE
deletion confirmation experience for screen

### PR DESCRIPTION
**Pull request by Bjvanminnen**
deletion

From the spec:
image

### CODE REVIEW

**Comment by davidsbailey** [comment]
onDone could also be removed from a few other places, now that it doesn't exist in the UI any more

File: apps/test/solutions/applab/ec_screens.js [hunk 1]
```diff
@@ -349,8 +361,10 @@ module.exports = {
 
         assert(button.parentNode === outerDiv);
 
+        var deleteButton = $("#design-properties button").eq(-1);
```
Comment (davidsbailey) [inline apps/test/solutions/applab/ec_screens.js]: Is there more than one #design-properties button in the DOM?

Comment (Bjvanminnen) [inline apps/test/solutions/applab/ec_screens.js]: Color picker rows end up with a button, possibly others as well.

Comment (davidsbailey) [inline apps/test/solutions/applab/ec_screens.js]: oh never mind, I see it is a descendant selector.

**Comment by davidsbailey** [comment]
LGTM

**Pull request merged**